nomos-da/full-replication/src/lib.rs

```diff
@@ -69,6 +71,7 @@ impl<A, C> AbsoluteNumber<A, C> {
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Settings {
+    pub private_key: [u8; 32],
```

Comment:
Why is this a private key here?

Reply:
What I mean is that I am not really sure this is the right move. If we are not signing anything here. I think we are not. Just a public key should be enough. is this used to issue the Voter?

Reply:
You're right. I guessed we're gonna make voter sign on the attestation in the future. But, if it's not (or if it's not determined yet), it may better to accept a `voter: [u8; 32]` (not even `public_key`).

Reply:
I'll change it and ping you again when it's ready.

Reply:
@danielSanchezQ Now, we don't involve the private key: 765d5a2d. We just accept a `voter` which is a `[u8; 32]` as a parameter. I think this is the simplest approach for now since we don't know yet what we're gonna do with this voter.